profiles/seccomp/default.json

```diff
@@ -186,6 +187,9 @@
 				"landlock_add_rule",
 				"landlock_create_ruleset",
 				"landlock_restrict_self",
+				"landlock_add_rule",
+				"landlock_create_ruleset",
+				"landlock_restrict_self",
```

Comment:
You can remove the landlock ones; those were just merged in https://github.com/moby/moby/pull/43199 (so are in there twice now 😅)

Reply:
Done!